src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,15 +1,144 @@
-import { Component } from 'react';
+import React, { Component } from 'react';
 
-type Props = {};
-type State = {};
+type Props = {
+  add: (movie: Movie) => void,
+};
+
+type State = {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+};
 
 export class NewMovie extends Component<Props, State> {
-  state: State = {};
+  state: State = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+
+    this.props.add({
+      ...this.state,
+    });
+
+    this.setState({
+      title: '',
+      description: '',
+      imgUrl: '',
+      imdbUrl: '',
+      imdbId: '',
+    });
+  };
 
   render() {
+    const {
+      title,
+      description,
+      imgUrl,
+      imdbUrl,
+      imdbId,
+    } = this.state;
+
     return (
-      <form>
-        Put the form here
+      <form onSubmit={this.handleSubmit}>
+        <div className="form-group">
+          <label htmlFor="title" className="col">
+            Title
+          </label>
+
+          <input
+            type="text"
+            id="title"
+            name="title"
+            value={title}
+            onChange={e => {
+              this.setState({ title: e.target.value });
+            }}
```

Comment:
багато раз повторюєшся зроби окремий метод  типу:
```suggestion
            handleChange={event: => {
            const { value, id} = event.target;
            
             this.setState({ [id]: value });
            }}
```
повинно спрацювати